portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase

```diff
@@ -0,0 +1,385 @@
+		ACUtils.clickAnyButton(button = "Revoke");
+
+		ACUtils.clickAnyButton(button = "Continue");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Feature ID: LRAC-8103 | Automation ID: LRAC-11502 | Test Summary: Assert the user is able to query data using Analytics Cloud API"
+	@priority = 3
+	test CreateQueryDataWithAnalyticsCloudAPI {
+		var typeList = "blogs,documents-and-media,forms,individuals,pages,segments,web-contents";
+
+		task ("Get the token value") {
+			var token = ACApi.getTokenValue();
+		}
+
+		task ("Create a query data using analytics cloud API") {
+			ACApi.createDataReport(token = ${token});
+		}
+
+		task ("Check the list of endpoints that are available for querying in the API") {
+			ACApi.viewApiResponseForRootEndpoint(typeList = ${typeList});
+		}
+	}
+
+	@description = "Feature ID: LRAC-12919 | Automation ID: LRAC-13511 | Test Summary: Check that an error message appears when using an invalid type in the endpoint"
+	@priority = 2
+	test RequestShowErrorMessageWhenUseInvalidType {
+		var typeList = "accounts,acqa";
+		var message = "The \"type\" query parameter must be either blogs, documents-and-media, forms, individuals, pages, segments, or web-contents.";
+
+		task ("Get the token value") {
+			var token = ACApi.getTokenValue();
+		}
+
+		for (var type : list ${typeList}) {
+			task ("Create a data report using analytics cloud API with invalid type") {
+				ACApi.createDataReport(
+					token = ${token},
+					type = ${type});
+			}
+
+			task ("Check if the API response is correct for the error case") {
+				ACApi.viewApiResponseForError(message = ${message});
+			}
+		}
+	}
+
+	@description = "Feature ID: LRAC-12919 | Automation ID: LRAC-13479 | Test Summary: Check that you can access data from all existing blogs"
+	@priority = 4
+	test RequestToAccessDataForAllBlogs {
+		var blogsTitleList = "Blog AC Title 1,Blog AC Title 2";
+		var type = "blogs";
+		var range = "rangeKey";
+		var rangeValue = 0;
+
+		task ("Copy the token value for the API") {
+			var token = ACApi.getTokenValue();
+		}
+
+		task ("Take the token from Analytics Cloud") {
+			ACSettings.takeTokenFromAnalyticsCloud();
+		}
+
+		task ("Go to DXP and create a site and two blogs and then connect Analytics Cloud to DXP") {
+			TestCase.setUpPortalInstance();
+
+			ACUtils.launchDXP();
+
+			ACUtils.addSiteAndPage();
+
+			for (var blogTitle : list ${blogsTitleList}) {
+				ACUtils.createBlogsAndAddToPage(entryTitle = ${blogTitle});
+			}
+
+			ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+		}
+
+		task ("Visit the two blogs") {
+			ACUtils.navigateToSitePage(
+				actionType = "View Blog",
+				documentTitleList = "Blog AC Title 1,Blog AC Title 2",
+				pageName = "AC Page",
+				siteName = "Site Name");
+
+			ACUtils.closeAllSessionsAndWait();
+		}
+
+		task ("Create a data report using analytics cloud API with the type blogs") {
+			var apiResponse = ACApi.createDataReport(
```

Comment:
Hey @manuferreira, `apiResponse` is a `static var` in `ACApi.createDataReport`, so you don't need to put a new variable to store the return value